src/Microsoft.AspNetCore.Session/CookieProtection.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using Microsoft.AspNetCore.DataProtection;
+
+namespace Microsoft.AspNetCore.Session
+{
+    public static class CookieProtection
```

Comment:
Was there a reason this needs to be public static class instead of just private helpers in the middleware?